Donor/Entities/DonationPreference.cs

```diff
@@ -0,0 +1,11 @@
+namespace Donor.Entities
+{
+    public class DonationPreference
```

Comment:
The relationship between Donor and Organ is a Many to Many relationship, if I'm not mistaken.
If that's the case the representation in .Net (EF 6.X) is through navigation properties.
**Therefore, in Donor you should have:** 
  public int  OrganId { get; set; }
  public ICollection<Organ> Organs{ get; set; }
In Organ you should have : 
  public int  DonorId { get; set; }
  public ICollection<Donor> Donors{ get; set; }

To sum up, the DonationPreference class is not needed as an entity in your code! But it is need it in SQL.